src/Microsoft.ML.Core/Utilities/Random.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.ML.Runtime
     public interface IRandom
     {
         /// <summary>
-        /// Generates a Single in the range [0, 1).
+        /// Generates a Float (single precision floating point) in the range [0, 1).
         /// </summary>
-        Single NextSingle();
+        Single NextFloat();
```

Comment:
>NextFloat [](start = 15, length = 9)

Hi @ thanks for the change.

I understand that it results in fewer files changed, but let's please not change `IRandom`. I want to get rid of the `RandomUtils`, but I'm less enthusiastic about making `IRandom` resemble it. Maybe use VS to rename this back to `NextSingle`. 

More generally: `Float` (capital `F`) is not a concept we want to expose. Back when we had "single" and "double" precision builds, with `Float` type aliases in practically every file it was a concept in this codebase, but those days are long over. Now then, the `RandomUtils`, being a shim to facilitate that, exposed this. (Once upon a time, there was an `#ifdef` surrounding that block of code you've now removed, plus a few alternate codepaths.) But we should be working on removing vestigates from that time rather than having them infect "clean" innocent interfaces that had nothing to do with that. :D

Reply:
Make sense. I will rename NextFloat() -> NextSingle().